src/main/java/uk/gov/hmcts/reform/professionalapi/controller/SuperController.java

```diff
@@ -324,7 +324,16 @@ protected ResponseEntity searchUsersByOrganisation(String organisationIdentifier
         return responseEntity;
     }
 
-    protected ResponseEntity<ModifyUserRolesResponse> modifyRolesForUserOfOrganisation(UserProfileUpdatedData userProfileUpdatedData, String userId, Optional<String> origin) {
+    protected ResponseEntity<ModifyUserRolesResponse> modifyRolesForUserOfOrganisation(UserProfileUpdatedData userProfileUpdatedData, String userId, Optional<String> origin, String orgId) {
+
```

Comment:
This validations are not required for the external controller endpoint.

Reply:
I will remove validation from here and place it in the Internal Controller before it goes to the Super Controller

Reply:
I have addressed this in the following commit: https://github.com/hmcts/rd-professional-api/pull/442/commits/532c5871ac1c709b24b0c72fe11d80c318308122